roles/ceph-osd/tasks/main.yml

```diff
@@ -27,6 +27,21 @@
   # Hard code this so we will skip the entire file instead of individual tasks (Default isn't Consistent)
   static: False
 
+- name: create lvm OSDs with ceph-volume
```

Comment:
To stay consistent with the playbook, please do not use any capital letters.
Also, the other includes don't have names, I'm not saying it's a good/bad idea I'm thinking about the consistency here :)